tests/test_guardrails_pii_detection.sh

```diff
@@ -25,11 +25,16 @@ function validate_microservice() {
     echo "Validate microservice started"
     export PATH="${HOME}/miniforge3/bin:$PATH"
     source activate
-    echo "test 1 - single task"
-    python comps/guardrails/pii_detection/test.py --test_text --batch_size 1 --ip_addr $ip_address
-    echo "test 2 - 20 tasks in parallel"
-    python comps/guardrails/pii_detection/test.py --test_text --batch_size 20 --ip_addr $ip_address
+    echo "test 1 - single task - ner"
+    python comps/guardrails/pii_detection/test.py --test_text --batch_size 1 --ip_addr $ip_address --strategy ner
+    echo "test 2 - 20 tasks in parallel - ner"
+    python comps/guardrails/pii_detection/test.py --test_text --batch_size 20 --ip_addr $ip_address --strategy ner
+    echo "test 3 - single task - ml"
+    python comps/guardrails/pii_detection/test.py --test_text --batch_size 1 --ip_addr $ip_address --strategy ml
+    echo "test 4 - 20 tasks in parallel - ml"
+    python comps/guardrails/pii_detection/test.py --test_text --batch_size 20 --ip_addr $ip_address --strategy ml
```

Comment:
three strategies were given but only two of strategies were tested

Reply:
Right now we only support two strategies: NER and ML.

Reply:
Deleted LLM strategy related code.